amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -175,6 +175,8 @@ type Query {
     @function(name: "getReputationForTopDomains")
   getUserReputation(input: GetUserReputationInput): String
     @function(name: "getUserReputation")
+  getNetworkInverseFee: String
+    @function(name: "getNetworkInverseFee")
```

Comment:
No no. This should not longer function as it did back in the Dapp.

This value is very rarely changed, even more so, when it gets changed an event gets emitted _(as far as I remember, check with @area)_

So this logic needs to be moved over in the block ingestor, fetch the value in there, stored in the database and retrieved properly via graphql queries.